Jenkinsfile

```
@@ -0,0 +1,42 @@
+#!groovy
+
+pipeline {
+
+    agent {
+        label 'general-purpose'
+    }
+    tools {
+        jdk "zulu-11"
+        maven "maven-3.6.3"
+    }
+    environment {
+        JAVA_HOME = tool("zulu-11")
+        MAVEN_OPTS = '-Xmx2G -Djavax.net.ssl.trustStore=${JAVA_HOME}/jre/lib/security/cacerts'
+        payaraBuildNumber = "${BUILD_NUMBER}"
+    }
+    stages {
+
+        stage('Checkout Payara6') {
+            steps {
+                script {
+                    checkout changelog: false, poll: true, scm: [$class: 'GitSCM',
+                    branches: [[name: "'deployment-transformer-1.x"]],
+                    doGenerateSubmoduleConfigurations: false,
+                    extensions: [], 
+                    submoduleCfg: [],
+                    userRemoteConfigs: [[credentialsId: 'payara-devops-github-personal-access-token-as-username-password', url:"https://github.com/payara/transformer.git"]]]
+                }
+            }
+        }
+        stage('Build') {
+            steps {
+                script {
+                    echo '*#*#*#*#*#*#*#*#*#*#*#*#  Building SRC  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
+                    sh """mvn -B -V -ff -e clean install --strict-checksums \
```

For this specific branch, we only want to build the deployment transformer extension
```suggestion
                    sh """mvn -B -V -ff -e clean install -f deployment-transformer/pom.xml --strict-checksums \
```